src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewAboutToSubmitHandlerTest.java

```diff
@@ -95,4 +112,75 @@ void shouldReturnWithoutError() {
 
         assertThat(response.getErrors()).isEmpty();
     }
+
+    @ParameterizedTest
+    @EnumSource(value = PostHearingReviewType.class, names = {"SET_ASIDE"})
+    public void givenPostHearingWithGivenReviewType_thenRespectiveDocTypeShouldBeAdded(PostHearingReviewType reviewType) {
+        switch (reviewType) {
+            case SET_ASIDE:
+                documentType = SET_ASIDE_APPLICATION;
+                break;
+            default:
+                documentType = null;
+        }
```

Comment:
You should avoid switch statements in tests. This leads to needing tests for your tests, which is a slippery slope and not what we want. 
A better approach here would be to change to CsvSource with two args: PostHearingReviewType and DocumentType. This way you know with the given input what you are expecting, and not rely on internal logic.